Chocolatey.Cake.Recipe/Content/install-module.ps1

```diff
@@ -0,0 +1,33 @@
+# Copyright © 2023 Chocolatey Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[cmdletBinding()]
+Param(
+    [Parameter()]
+    [String]
+    $ModuleName,
+
+    [Parameter()]
+    [String]
+    $RequiredVersion
+)
+
+if (Get-Module -ListAvailable -Name $ModuleName) {
+    Write-Host "The $ModuleName PowerShell Module is already installed."
```

Comment:
This probably should check the version of the module as well, assuming you want to lock the build into a specific version or a specific minimum version.

Reply:
That is a good idea!  I will get that added.